deployments/ansible/molecule/default/Dockerfile.j2

```diff
@@ -28,7 +28,7 @@ RUN zypper -n clean && zypper -n refresh
 RUN zypper -n install -l ansible dbus-1 rpm-python sudo systemd-sysvinit
 {% else %}
 FROM opensuse/leap:15
-RUN sed -i 's|download.opensuse.org|provo-mirror.opensuse.org|' /etc/zypp/repos.d/*.repo
+#RUN sed -i 's|download.opensuse.org|provo-mirror.opensuse.org|' /etc/zypp/repos.d/*.repo
```

Comment:
Why commented?

Reply:
the provo mirror is broken. Jina recommended removing it